setup.py

```diff
@@ -38,7 +38,7 @@
             "sentry-sdk",
         ],
         # Data validation
-        "validation": ["schematics"],
+        "validation": ["schematics", "cerberus"],
```

Comment:
Will people using validation usually need code that depends on both packages, or code that depends on one of them? Should we change how we handle these optional dependencies?

Reply:
Indeed, probably it would be better to install them separately

Reply:
+1 to that.

Reply:
I see this line is still unchanged. An oversight?